<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CE5UserGroup</class>
 <widget class="QWidget" name="CE5UserGroup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Description</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Code</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="C5LineEditWithSelector" name="leCode">
         <property name="Field" stdset="0">
          <string>f_id</string>
         </property>
         <property name="Primary" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="C5LineEditWithSelector" name="lineEdit_2">
         <property name="Field" stdset="0">
          <string>f_name</string>
         </property>
         <property name="Type" stdset="0">
          <string>0</string>
         </property>
         <property name="Check" stdset="0">
          <string>empty=&quot;Name&quot;</string>
         </property>
         <property name="First" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Start time</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="C5LineEditWithSelector" name="leStartTime">
         <property name="Field" stdset="0">
          <string>f_starttime</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Duration</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="C5LineEditWithSelector" name="leDuration">
         <property name="Field" stdset="0">
          <string>f_duration</string>
         </property>
         <property name="Type" stdset="0">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Salary</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="3" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Value of department</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="C5LineEdit" name="leFixed"/>
       </item>
       <item row="1" column="1">
        <widget class="C5LineEdit" name="leMin"/>
       </item>
       <item row="6" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Maximum</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Value of total</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="C5LineEdit" name="leValTotal"/>
       </item>
       <item row="2" column="1">
        <widget class="C5LineEdit" name="leMax"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Minimum</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Fixed</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="C5LineEdit" name="leValDep"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Own total</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="C5LineEdit" name="lwOwnTotal"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>C5LineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">c5lineedit.h</header>
  </customwidget>
  <customwidget>
   <class>C5LineEditWithSelector</class>
   <extends>QLineEdit</extends>
   <header location="global">c5lineeditwithselector.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
